exporter/clickhouseexporter/exporter_traces.go

```diff
@@ -103,7 +104,7 @@ func (e *tracesExporter) pushTraceData(ctx context.Context, td ptrace.Traces) er
 						r.Name(),
 						traceutil.SpanKindStr(r.Kind()),
 						serviceName,
-						resAttr,
+						attributesToMap(resAttr, scopeAttr),
```

Comment:
you could do this once per scope, right?

Reply:
yes